src/mpi-rt/controllers/SpotController.cpp

```diff
@@ -232,6 +232,9 @@ class SpotController : public cali::ChannelController
             if (opts.get("output", "").to_string() != "adiak")
                 config()["CALI_SERVICES_ENABLE"].append(",adiak_import");
 #endif
+            if (opts.get("profile.kokkos","").to_bool()) {
+                config()["CALI_SERVICES_ENABLE"].append(",kokkostime");
+            }
             m_opts.update_channel_config(config());
```

Comment:
Won't need that, the options take care of the services automatically

Reply:
Addressed